src/DataStructures/DataBox/Prefixes.hpp

```diff
@@ -100,4 +100,27 @@ struct Next : db::PrefixTag, db::SimpleTag {
   using tag = Tag;
   static std::string name() noexcept { return "Next(" + Tag::name() + ")"; }
 };
+
+/// \ingroup DataBoxTagsGroup
+/// \brief Prefix corresponding to the characteristic speed of the field given
+/// by `Tag`. `type` is always a `Scalar<DataVector>`.
+template<typename Tag>
+struct CharSpeed : db::PrefixTag, db::SimpleTag {
+  using type = Scalar<DataVector>;
+  using tag = Tag;
+  static std::string name() noexcept {
+    return "CharSpeed(" + Tag::name() + ")";
+  }
+};
+
+/// \ingroup DataBoxTagsGroup
+/// \brief Prefix corresponding to the characteristic field given by `Tag`.
+template<typename Tag>
+struct Field : db::PrefixTag, db::SimpleTag {
+  using type = typename Tag::type;
+  using tag = Tag;
+  static std::string name() noexcept {
+    return "Field(" + Tag::name() + ")";
+  }
```

Comment:
Newbie question: is this convention, including parentheses, the right way to name tags? I would have expected GeneralizedHarmonic::Field::TagName or something like that.

Reply:
I also just went off of what was done for the other Prefix tags, which were added by Will I think so I was assuming that was our convention. 

However, I'm not sure the `Field` prefix is needed, since the tags `Psi`, `Pi`, `Phi`, `UPsi`, `UZero`, `UPlus`, `UMinus` already represent fields. The reason I added the `CharSpeed` prefix was so that for a tag `U`, `CharSpeed<U>` would represent the characteristic speed corresponding to the field `U`, and so that `CharSpeed<U>::type` would be `Scalar` regardless of the type of `U::type`, a whereas there isn't really a difference between `U` and `Field<U>` in their usage. 

Unless people think that `Field` explicitly tags something as being a characteristic field makes it clearer, - or I've missed something - I'd vote to not use the prefix tag, but I'll defer to @kidder, @nilsdeppe, @wthrowe,

Reply:
I agree that Field shouldn't be necessary, but I'd defer to the consensus.

Reply:
I originally thought it to be a code convention that the returned `Variables` container contain all fields tagged by structs of the _same type_ (by wrapping `Psi`, `Pi`, `Phi`, `UPsi`, `UPlus`, `UMinus`, and `UZero` in `Field`), which would have made sense if (such) tagging structs were being used across evolution systems. Since that is clearly not the case, I completely agree that there is no real benefit of wrapping in `Field`. I have removed it.